src/app/integrations/edit-page/step-configure/data.mapper.example.host.component.ts

```diff
@@ -77,21 +78,26 @@ export class DataMapperHostComponent extends FlowPage implements OnInit, OnDestr
     switch (event.kind) {
       case 'integrations-mapper-init':
         const step = this.currentFlow.getStep(this.position);
+        let mappings = undefined;
         if (step.configuredProperties && step.configuredProperties[MAPPING_KEY]) {
           try {
-            this.cfg.mappings = <any> JSON.parse(step.configuredProperties[MAPPING_KEY]);
+            mappings = <any> JSON.parse(step.configuredProperties[MAPPING_KEY])['AtlasMapping'];
           } catch (err) {
             // TODO
           }
         }
-        if (!this.cfg.mappings) {
-          this.cfg.mappings = new MappingDefinition();
+        this.cfg.mappings = new MappingDefinition();
+        console.log('Mappings: ', mappings);
```

Comment:
rofl, yeah, looked at my editor and realized I hadn't saved the file before committing, darnit!